modules/ztp-filtering-ai-crs-using-siteconfig.adoc

```diff
@@ -0,0 +1,89 @@
+// Module included in the following assemblies:
+//
+// scalability_and_performance/ztp-deploying-disconnected.adoc
+
+:_module-type: PROCEDURE
+[id="ztp-filtering-ai-crs-using-siteconfig_{context}"]
+= Filtering custom resources using SiteConfig filters
+
+Using filters, you can easily customize `SiteConfig` custom resources (CRs) to include or exclude other CRs for use in the installation phase of the zero touch provisioning (ZTP) GitOps pipeline.
+
+The following example configures a custom single-node cluster `SiteConfig` CR that excludes the `siteconfig/extra-manifest/03-sctp-machine-config-worker.yaml` file from the installation.
+
+.Prerequisites
+
+. You configured the hub cluster for generating the required installation and policy CRs.
```

Comment:
Prereqs are an unordered list, not an ordered list. ([procedure](https://raw.githubusercontent.com/redhat-documentation/modular-docs/master/modular-docs-manual/files/TEMPLATE_PROCEDURE_doing-one-procedure.adoc) template)